bigquery2/bigquery_constants.bal

```diff
@@ -30,9 +30,36 @@ const string QUESTION_MARK = "?";
 const string AND_SIGN = "&";
 const string SLASH = "/";
 
+const string TOKEN_ENDPOINT = "/oauth2/v4/token";
+const string JWT_HEADER_ALGO_VALUE = "RS256";
+const string JWT_HEADER_TYPE_VALUE = "JWT";
+const string GRANT_TYPE_HEADER = "urn:ietf:params:oauth:grant-type:jwt-bearer";
+const string SCOPE = "scope";
+const string PASSWORD = "notasecret";
+const string KEYALIAS = "privatekey";
+
 public const POSITIONAL_MODE = "POSITIONAL";
 public const NAMED_MODE = "NAMED";
 public type ParameterMode "POSITIONAL"|"NAMED";
 
 // Error Codes.
-final string BIGQUERY_ERROR_CODE = "(wso2/bigquery2)BigqueryError";
\ No newline at end of file
+final string BIGQUERY_ERROR_CODE = "(wso2/bigquery2)BigqueryError";
```

Comment:
This can also be a constant right? 

And why are we using parentheses instead of curly braces? Can we change to 
```suggestion
const BIGQUERY_ERROR_CODE = "{wso2/bigquery2}BigqueryError";
```

Reply:
Also, the name of it is "BigQuery", not "Bigquery"